src/main/java/Command.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * The Command class handles the validation of a supported command and printing of its usage.
+ */
+public class Command {
+
+    /* The supported command */
+    private String command;
+
+    /**
+     * Instantiates a new supported command.
+     *
+     * @param command Command to support.
+     */
+    public Command(String command) {
+        this.command = command;
+    }
+
+    public String getCommand() {
+        return command;
+    }
+
+    /**
+     * Checks if the given input is indicating that the user is attempting to run this command.
+     *
+     * @param inputLine Raw input line to check.
+     * @return True if the user is trying to run this command.
+     */
+    public boolean isCommand(String inputLine) {
+        String commandLine = inputLine.strip();
+        return commandLine.equals(command) || commandLine.startsWith(command + " ");
+    }
+
+    /**
+     * Checks if the given input is a valid command.
+     *
+     * @param inputLine Raw input line to check.
+     * @return True if the input is valid.
+     */
+    public boolean isValidCommandLine(String inputLine) {

```

Comment:
It is a good practice to have "isValidCommandLine" as the name of this method, it is quite clear and understandable.